webui-ros-joystick/launch/webui.launch

```diff
@@ -2,19 +2,21 @@
 
     <arg name="node_name" default="node_server" />
     <arg name="wait_nodes" default="" />
```

Comment:
```suggestion
    <arg name="await_nodes" default="" />
```

Reply:
Old naming, would require changes in few files